src/features/landing/components/ForWhoSection/ForWhoSectionCards/ForWhoSectionCards.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, Grid, Typography } from '@mui/material';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import useScreenSize from '../../../hooks/useScreenSize/useScreenSize';
+import { TRANSLATION_NAMESPACE } from '../../../utils/constants/constants';
+
+const styles = {
+  mainBox: {
+    position: 'relative',
+    top: '-13.4375rem', // -215px
+    padding: '0 0 0 0',
```

Comment:
for what reason you write padding like this? you can write just like this 'padding: 0'